scripts/models/TownHall.js

```diff
@@ -111,6 +111,17 @@
     }, data).sort(TownHall.sortFunction);
   };
 
+  // TODO: replace this with a flag on the model from the server
+  TownHall.isStateEvent = function(townhall) {
+    // Some state events don't seem to have the "District" value
```

Comment:
a better check is thp_id, eventually none of the events will have `District`. in the long run I am going to put them at their own endpoint.

Reply:
gotcha.  Is it safe to assume if an object does not have a thp_id then it's not a state event for now?

Reply:
yeah